CIP-XXXX/README.md

```diff
@@ -0,0 +1,506 @@
+---
+CIP: XXX
+Title: Own-data Wallet Extension
+Category: Tools
+Status: Proposed
+Authors:
+  - Giovanni Garufi <giovanni@mlabs.city>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/957
+Created: 2024-02-05
+License: CC-BY-4.0
+Solution-To: CPS-0010
```

Comment:
```suggestion
```
Although candidate CIP-0144 _is_ definitely a solution to CPS-0010, I would argue that this CIP is not... because it's an extension to an API rather than an API itself.  Again, this may have been inadvertently copied... but if not, I would argue that including links to indirectly related CPSs could make the front matter detail generally longer and more difficult to maintain and navigate.

Reply:
resolved with https://github.com/cardano-foundation/CIPs/pull/986/commits/58875ec56a9da37c8c0e092a905be99bd0993e72